nuget/Microsoft.Windows.CsWinRT.nuspec

```diff
@@ -34,5 +34,7 @@
     <file src="$winrt_host_arm$" target ="runtimes\win-arm\native"/>
     <file src="$winrt_host_arm64$" target ="runtimes\win-arm64\native"/>
     <file src="$winrt_shim$" target ="lib\net5.0\"/>
+    <file src="$guid_patch$" target ="build\tools\"/>
+    <file src="$net5_runtime$" target ="build\tools\"/>
```

Comment:
Are we copying winrt.runtime also to to the tools folder?  Can we just use the one from the lib\net5.0 folder?